.circleci/config.yml

```diff
@@ -67,6 +67,7 @@ jobs:
       - checkout
       - attach_workspace:
           at: /go/src/github.com/gruntwork-io/cloud-nuke
+      - run: cd bin && sha256sum * > SHA256SUMS
```

Comment:
Wouldn't this be better as `sha256sum bin/* > bin/SHA256SUMS`? I'm not sure if each `run` in CircleCI is executed in its own shell instance because if it isn't the below command will fail as there'll be no `bin` folder in the `bin` folder

Reply:
This https://github.com/gruntwork-io/health-checker/pull/14#issuecomment-469244605 answers my question